sktime/classification/shapelet_based/_rocket_classifier.py

```diff
@@ -28,14 +34,15 @@ class ROCKETClassifier(BaseClassifier):
     ----------
     num_kernels             : int, number of kernels for ROCKET transform
     (default=10,000)
-    ensemble                : boolean, create ensemble of ROCKET's (default=False)
-    ensemble_size           : int, size of the ensemble (default=25)
+    n_estimators            : int, ensemble size, optional (default=None)
     random_state            : int or None, seed for random, integer,
     optional (default to no seed)
+    n_jobs                  : int, the number of jobs to run in parallel for `fit`,
```

Comment:
Do you also want to add parallelisation to `predict`?

Reply:
I'm not sure about that, predict is straightforward and reasonably fast (at least when compared to fitting). I don't know if the overhead of `Parallel` is even worth it. If we do want to parallelize predict as well, it might be good to have different `n_jobs` for this, e.g. I have `n_jobs=25` for fitting and `n_predict_jobs=2` for predict. Or... maybe an additional parameter for `predict` call itself? What do you think?

Reply:
Okay let's leave it at fit for now